pkg/policy/l4.go

```diff
@@ -71,13 +71,19 @@ func (l7 L7DataMap) MarshalJSON() ([]byte, error) {
 // defines all supported types of L7 parsers
 type L7ParserType string
 
+func (l7 *L7ParserType) String() string {
```

Comment:
Remove the pointer from `l7 *L7ParserType`. The `String() string` interface will work for both cases

```golang
l1 := L7ParserType("foo")
l2 := L7ParserType("foo")
fmt.Printf("%s\n", l1) // foo
fmt.Printf("%s\n", &l2) // foo
```

Reply:
Removed.